apps/site/lib/site_web/controllers/schedule/line/helpers.ex

```diff
@@ -121,7 +121,6 @@ defmodule SiteWeb.ScheduleController.Line.Helpers do
     conn
```

Comment:
@amaisano it's OK if this goes in this PR, although the change should certainly stay in its own commit. And that commit should be early in the PR, in keeping with the general principle that first you fix stuff, then you build new stuff.